### PR TITLE
Cloud Secrets Documentation

### PR DESCRIPTION
**Pull request by kraftp**

### CODE REVIEW

File: docs/cloud-tutorials/cicd.md [hunk 1]
```diff
@@ -26,18 +26,21 @@ There's nothing special about the `-staging` and `-prod` suffixes&mdash;you can
...
+You can also create or update a secret using the cloud CLI:
+
+```
+dbos-cloud app secrets create -s <secret-name> -v <secret-value>
 ```
 
-Then the value of `API_KEY` in your client environment will be submitted with your application and supplied to it as an environment variable on DBOS Cloud.
-Only environment variables declared in `dbos-config.yaml` are submitted and supplied like this.
+Secrets are made available to your application as environment variables.
+The name of the environment variable is your secret name, its value is your secret's value.
```
Comment (qianl15) [inline docs/cloud-tutorials/cicd.md:43]: What if someone updates a secret -- will it only affect new deploys or will it restart all VMs? It's good to clarify the behavior here.

File: docs/cloud-tutorials/cloud-cli.md [hunk 2]
```diff
@@ -321,6 +321,28 @@ It is meant to be used with [`database restore`](#dbos-cloud-db-restore) during
 
 ---
 
+### `dbos-cloud app secrets create`
+
+**Description:**
+Create a new secret associated with an application, or update an existing secret.
```
Comment (qianl15) [inline docs/cloud-tutorials/cloud-cli.md:327]: Similar comment as above. It'd be great to clarify the behavior of secret updates.

File: docs/cloud-tutorials/cicd.md (hunk 1 above)
Comment (kraftp) [reply docs/cloud-tutorials/cicd.md:43]: Done!

**Review by apoliakov — APPROVED** [review]
I dunno if you intended to get reviews from others as well but it looks great to me.

**Pull request merged**